devops/openshift/forms-flow-ai/bpm-deploy.yml

```diff
@@ -107,6 +126,8 @@ objects:
                       name: ${FORMIO_SECRET}
                 - name: DEBUG
                   value: "False"
+                - name: MAIL_CONFIG
```

Comment:
name can be EMAIL_CONFIG instead of MAIL because we may have to use config related to physical mail(in future).

Reply:
As the actual file name of the configuration is like /app/mail-config.properties. Its used this way. Not sure why they used it in camunda 👍